mirascope/cli/generic/mirascope.ini.j2

```diff
@@ -11,3 +11,7 @@ prompts_location = {{ prompts_location }}
 
 # name of versions text file
 version_file_name = {{ version_file_name }}
+
+# formats the version file    
+# defaults to ruff format
+# format_command = 
```

Comment:
why not just default this to ["ruff", "format"] and clean up the code to always require format command?